src/intent.html

```diff
@@ -347,37 +287,76 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
-   structure can be supplied by the AT system, as it would for an
-   un-annotated table.</p>
+   <h3 id="intent_using">Using Intent Concepts and Properties</h3>
+   <p>When the <code class="attribute">intent</code> attribute corresponding to a specific node
+     contains a concept component, possibly with properties and arguments,
+     the AT's [=Intent Concept Dictionary=] should be consulted.
+     The concept name should be normalized
+     (<q>`_`</q> (U+00F5) and <q>`.`</q>  (U+002E) to <q>`-`</q> (U+002D)),
+     and compared using <a data-cite="INFRA#ascii-case-insensitive">ASCII case-insensitive</a>
+     match.</p>
+   <p>An concept is considered a <dfn id="intent_known_concept">known concept</dfn> (to the AT)
+     when the normalized name,
+     the fixity property (which may be defaulted in the concept dictionary),
+     and the arity (number of arguments, if any)
+     all match an entry in the AT's concept dictionary.
+     The speech hint in the matching entry
+     can be used be used as a guide for the generation of
+     specific audio, replacement text or braille renderings, as appropriate.
+     However, because common notations have many specialized ways of being spoken, the AT
+     is NOT constrained to use the hint as given.  For example, AT may
+     vocalize a fraction marked up with <code>&lt;mfrac></code>
+     as <q>three quarters</q> or <q>three over x</q>
+     or may vocalize an inline fraction marked up as <code>&lt;mo>/&lt;/mo></code>
+     as <q>three divided by x</q>.
+     The choice may depend on the contents
+     and carrier element associated with an
+     <code>intent="divide($num,$denom)"</code>.
+     Note that properties other than those specifying fixity
+     may also indicate different rendering choices.</p>
+   <p>Otherwise, if the concept name, fixity and arity do not match it is considered to be an
+     <dfn id="intent_unknown_concept">unknown concept</dfn> (to the AT)
+     and will be treated the same as a [=literal=];
+     that is, the name is spoken as-is after normalizing each of `-`, `_` and `.` to an inter-word space.
+     Even for an unknown concept, if a fixity property and arguments were given (or inferred),
+     the speech for the arguments should be composed
+     in a manner consistent with the given fixity property, if possible.</p>
+   <p>Note that future updates of the AT and [=Intent Concept Dictionary=] may
+     include additional concepts, at which time those concepts may also receive special treatment.</p>
+
+   <p><em>NEEDS CLARIFICATION: </em>
+     In cases where the intent is missing or does not contain an explicit concept,
+     or does not contain an explicitly structured intent with concept and arguments,
+     the AT may be able to infer the relevant concept. fixity and arguments
+     from the structure and content of the MathML.
+     In such cases, it should proceed as above as if the inferred intent had been given explicitly.
+   </p>
```

Comment:
```suggestion
   For example an AT system might infer  in intent of `power` given `<msup><mi>x</mi><mn>2</mn></msup>`,
   resulting in a reading such as "x squared". An explicit intent property of `:literal`  could be used to over-ride this inferred intent, and lead to a reading such as `x superscript 2`.
  </p>
```

Reply:
Right, could add that. But there's also the case of "short-hand" intent (if that's the right term): I was also concerned with a case like
```<mrow><mi>x</mi><mo intent="plus">foo</mo><mi>y</mi></mrow>```
which could be inferred to be
```<mrow intent="plus:infix($x,$y)">...```
Should this be addressed here? (or is it addressed elsewhere already?)

Reply:
I just pushed a *very* dubious section as a starting point for discussion.